handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -1131,6 +1141,15 @@ protected static PrivateKey toPrivateKey(InputStream keyInputStream, String keyP
         if (keyInputStream == null) {
             return null;
         }
+
+        // try BC first, if this fail fallback to original key extraction process
+        if (BouncyCastlePemReader.isAvailable()) {
+            PrivateKey pk;
+            if ((pk = BouncyCastlePemReader.getPrivateKey(keyInputStream, keyPassword)) != null) {
+                return pk;
+            }
+        }
+
```

Comment:
```suggestion
        if (BouncyCastlePemReader.isAvailable()) {
            PrivateKey pk = BouncyCastlePemReader.getPrivateKey(keyInputStream, keyPassword);
            if (pk != null) {
                return pk;
            }
        }
```

Reply:
+1 I am not a fan of inline assignments as they make the code harder to read